tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-e2e-test-nightly-pipeline.yml

```diff
@@ -0,0 +1,253 @@
+trigger: none
+
+jobs:
+- job: Onnxruntime_Linux_GPU_Training_E2E_Test_Nightly
+
+  timeoutInMinutes: 120
+
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+
+  # update these if the E2E test data changes
+  - script: |
+      orttraining/tools/ci_test/download_azure_blob.py \
+      --azure_blob_url https://onnxruntimetestdata.blob.core.windows.net/training/onnxruntime_training_data.zip?snapshot=2020-06-15T23:17:35.8314853Z \
+      --target_dir $(Build.BinariesDirectory)/training_e2e_test_data \
+      --archive_sha256_digest B01C169B6550D1A0A6F1B4E2F34AE2A8714B52DBB70AC04DA85D371F691BDFF9
+    displayName: 'Download onnxruntime_training_data.zip data'
+
+  - script: |
+      orttraining/tools/ci_test/download_azure_blob.py \
+      --azure_blob_url https://onnxruntimetestdata.blob.core.windows.net/training/glue_MRPC_data.zip \
+      --target_dir /bert_data/hf_data/
+    displayName: 'Download glue_MRPC_data.zip data'
+
+  - script: |
+      tools/ci_build/github/linux/run_dockerbuild.sh \
+        -o ubuntu16.04 -d gpu -r $(Build.BinariesDirectory) \
+        -x " \
+          --config RelWithDebInfo \
+          --enable_training \
+          --update --build \
+          --build_wheel \
+          --enable_training_python_frontend_e2e_tests \
+          --enable_training_pipeline_e2e_tests \
+          "
+    displayName: 'Build'
+
+  # Hit OOM with run_training_pipeline_e2e_tests.py - slightly above 16GB limit.
+  # leave this code here for further investigation.
+  # https://msdata.visualstudio.com/Vienna/_workitems/edit/956642
+  # - script: |
+  #     docker run \
+  #       --gpus all \
+  #       --shm-size=1024m \
+  #       --rm \
+  #       --volume $(Build.BinariesDirectory):/build \
+  #       --volume /bert_data:/bert_data \
+  #       --volume /bert_ort:/bert_ort \
+  #       onnxruntime-ubuntu16.04-cuda10.1-cudnn7.6 \
+  #         /build/RelWithDebInfo/run_training_pipeline_e2e_tests.py \
+  #           --cwd /build/RelWithDebInfo
+  #   displayName: 'Run run_training_pipeline_e2e_tests.py'
+  #   condition: succeededOrFailed() # ensure all tests are run
+  #   timeoutInMinutes: 10
```

Comment:
```suggestion
```

Reply:
Discussed with Xueyun. She is to make run_training_pipeline_e2e_tests.py pass and uncomment this block after merge of this PR.